docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -65,12 +67,23 @@ are not familiar with GitHub you can use this
 `tutorial <https://guides.github.com/activities/hello-world/>`__ to set up
 an account and repository.
 
+Creating a .dockstore.yml
+-------------------------
+
+You will need a ``.dockstore.yml`` file. It is used in conjunction with the :doc:`Dockstore GitHub App <github-apps/github-apps-landing-page>` to integrate your workflow with Dockstore. For other workflow languages, the ``.dockstore.yml`` is created in a text editor; for Galaxy you have the option of generating with a command-line tool, and it's recommended you do so.
```

Comment:
Borderline nit, but we might be able to condense the first two sentences into one that is more plain English ("in conjunction with" is a bit stilted, imho) and explains the purpose of `.dockstore.yml` and how our App uses it process the repo.

Reply:
Stilted, dost thou sayest? :) Fair point, I'll try to come up with something better.

The link in the second sentence is to our main documentation on how GitHub and .dockstore.yml work. I didn't want to duplicate that nor rewrite it here. On the other hand, we are asking the user to generate the file here, without having read that portion yet, so I'll try to come up with a little more detail.